frontend/src/metabase/visualizations/lib/action.js

```diff
@@ -13,10 +14,16 @@ export function performAction(
     const reduxAction = action.action();
     if (reduxAction) {
       dispatch(reduxAction);
+
       didPerform = true;
     }
   }
   if (action.url) {
+    // (metabase#51099) disable url click behavior
```

Comment:
This change still makes the table display the custom link text, and renders the column as a link.

https://github.com/user-attachments/assets/d46418e9-1af1-41c0-9bb9-fb2a3b4c5570


I haven't looked too much into the details but it seems like to address that we need to change formatValueRaw, specifically https://github.com/metabase/metabase/blob/e029869dfdf1517f7c38a5cc1f1c21df24f0e005/frontend/src/metabase/lib/formatting/value.tsx#L166-L167

Do you know how much we'd be in prod with this simpler solution?

Reply:
@npretto Good catch, I'll also update the table formatting! I think this will be in production until we can implement click behaviors (e.g. linking to questions and dashboards) for SDK: https://metaboat.slack.com/archives/C06FCQT0KMZ/p1733768624642749

Reply:
updated in https://github.com/metabase/metabase/pull/51290/commits/b321652d228fffb61a97ed184795e6c049ad3934, should be good to go now. Thanks!

![CleanShot 2567-12-20 at 03 57 00@2x](https://github.com/user-attachments/assets/38d1f5b6-0088-4000-a8d4-976d25891ecd)

Reply:
I was stil able to reproduce the issue for both my customized columns:
<img width="480" alt="image" src="https://github.com/user-attachments/assets/3fc8dbfa-4083-4094-aca2-9167744b4aad" />
<img width="1352" alt="image" src="https://github.com/user-attachments/assets/1f7a10a6-e04a-48dd-90d6-398e695c1240" />

Reply:
Note that I did customize how the link was rendered
<img width="860" alt="image" src="https://github.com/user-attachments/assets/dc02afeb-8665-41b6-8e51-71acc2ad467e" />

Reply:
@npretto Woah, I didn't knew that "On-click behavior for each column" was a thing - thanks! Was scouring the docs but couldn't find out how to set that up. Let me try that.

Reply:
updated the formatter to skip rendering custom link text and do not show those as links.

In the SDK's Storybook:

![CleanShot 2567-12-21 at 04 09 16@2x](https://github.com/user-attachments/assets/d8298cc0-9df8-4d9f-bd62-1be33ffd6a7f)

In the main app:

![CleanShot 2567-12-21 at 04 09 07@2x](https://github.com/user-attachments/assets/64e69f34-41e8-463c-9b57-d09a7041d13c)

Reply:
@npretto Thanks for pointing out the click behavior in tables! I updated the formatter to skip rendering custom link text and do not show those as links.

In the SDK's Storybook:

![CleanShot 2567-12-21 at 04 18 08@2x](https://github.com/user-attachments/assets/55b72c65-14d4-46b6-97ae-192336519787)

In the main app:

![CleanShot 2567-12-21 at 04 09 07@2x](https://github.com/user-attachments/assets/64e69f34-41e8-463c-9b57-d09a7041d13c)

PS. I wanted to add more E2E tests to cover these cases, but seems like it'll take a bit to setup and might not be worth it for a temporary solution. I might give it a try on Monday.